.travis.yml

```diff
@@ -1,12 +1,16 @@
 language: python
 python:
+  - '2.7'
+  - '3.4'
+  - '3.5'
   - '3.6'
```

Comment:
What's the story on 3.7 (and maybe [3.8!](https://docs.python.org/3.8/whatsnew/3.8.html)) support on travis?

Reply:
3.7 will be supported in another ticket. This one is simply just to introduce the migration to travis CI.